### DISCUSSION TITLE
Neovide Logo Suggestion based on Neovim and Rust Logos

### DISCUSSION THREAD

**Discussion by chrisbouchard** [neovide/neovide#515] [Show and tell]
I played around a bit with the Neovim and Rust logos in Inkscape to see if I could come up with anything reasonable to suggest, based on the conversation in [this issue](https://github.com/Kethku/neovide/issues/31). Here's what I came up with:

![Neovide Logo Suggestion](https://user-images.githubusercontent.com/4512028/113657054-2452b800-966b-11eb-8550-ba0f87295256.png)

I tried several ways of embedding the Neovim logo _in_ the Rust ring (both color and black-and-white variants), but I felt like the Neovim logo got lost. I feel like the Neovim part of the logo has to stand out stronger than the Rust part.

This image is shared under the [Creative Commons Attribution License (CC-BY)](https://creativecommons.org/licenses/by/4.0/). The Rust logo is &copy; Mozilla and used [according to their copyright and trademark licenses](https://www.rust-lang.org/policies/media-guide) (CC-BY). The Neovim logo is &copy; Jason Long (CC-BY 3.0).

**Comment by chrisbouchard** [comment]
Here are a couple iterations on the theme, this time with the Neovim logo embedded in the Rust ring.

![Neovide Logo Suggestion Embedded](https://user-images.githubusercontent.com/4512028/113658832-cd4ee200-966e-11eb-8cb3-0e6de88a3e3c.png)

**Reply by SaladinoBelisario** [reply]
I like the first one, less mental effort to visualize :)

**Comment by chrisbouchard** [comment]
Ok, one more variation and then I'm done for the night. It's based on the first one, but with the Rust ring intertwined with the Neovim logo.

![Neovide Logo Suggestion Intertwined](https://user-images.githubusercontent.com/4512028/113663810-af867a80-9678-11eb-8ef7-507be7225642.png)

**Reply by sudormrfbin** [reply]
This is simply a suggestion; how about Rust's ring going _over_ the Neovim's slant in the the bottom right corner ? So on the top left the ring would be under the _N_ and on the bottom right the ring would be over the _N_. Or the other way around :P

**Reply by chrisbouchard** [reply]
@gokulsoumya: I gave your idea a try. Inkscape is giving me a bit of trouble, so I was only able to get the bottom corner under so far, but here are some more variants based on your suggestion:

![Neovide Logo Suggestion Intertwined 2](https://user-images.githubusercontent.com/4512028/113801711-65100700-9727-11eb-8b45-62700aeee7fe.png)

(Please ignore the breaks in the gear. Those would presumably be cleaned up in a final copy.)

**Reply by Kethku** [reply]
Oh wow these are incredible! I'm absolutely no designer, but these look amazing. In particular the first one in the most recent set (with the bottom of the N under the ring) feels great! Feels like it highlights the V in the neovim N which is cool. Would you be willing to make a PR committing a version of these into the repo somewhere?

**Reply by chrisbouchard** [reply]
Absolutely. I'll work on a final version tonight.

**Comment by chrisbouchard** [comment]
For anyone following along later, we added a logo in [!519](/Kethku/neovide/pull/519).

**Comment by cpea2506** [comment]
Can we have it with MacOS Big Sur style? It has the base with all four corner rounded and the logo is in the center, like this:

![256](https://user-images.githubusercontent.com/42694704/146880984-1d00c55d-b1e4-4109-9a3a-257f6775dd24.png)

The original logo didn't look consistently with all the MacOS app's logo design since MacOS 11.

**Reply by JryChn** [reply]
support
